shm/Store.cpp

```diff
@@ -0,0 +1,23 @@
+#include "Store.hpp"
+
+Response Store::buy(Cargo* cargo, size_t amount, Player* player){
+    const size_t price = amount * cargo->getBasePrice();
+    if (cargo -> getAmount() < amount){
+        return Response::lack_of_cargo;
+    }
+    if (player -> getMoney() < price){
+        return Response::lack_of_money;
+    }
+    if (player -> getAvaliableSpace() < amount){
+        return Response::lack_of_space;
+    }
+    player->buyCargo(cargo, amount, price);
+    return Response::done;
+}
+
+Response Store::sell(Cargo* cargo, size_t amount, Player* player){
+    const size_t price = amount * cargo->getBasePrice();
```

Comment:
I don't think we would want to sell cargo for base price. It makes item rarity, alcohol strength obsolete

Reply:
+1, right.